### PR TITLE
Enhancement for "IP Enrichment - External - Generic v2" playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added a new task for "ip (VirusTotal (API v3))" Automation and changed the current "Get IP report from VirusTotal Private API" task to skip if automation/playbook is unavailable.

## Screenshots
"IP Enrichment - External - Generic v2" playbook:
IP_Enrichment_-_External_-_Generic_v2_copy_Wed_Nov_16_2022

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml [hunk 1]
```diff
@@ -601,6 +669,47 @@ tasks:
...
+  "44":
+    id: "44"
+    taskid: 3607b68d-96db-4ed1-8324-e21009548cf8
+    type: regular
+    task:
+      id: 3607b68d-96db-4ed1-8324-e21009548cf8
+      version: -1
+      name: Check the IP reputation
+      description: Checks the reputation of an IP address.
+      script: VirusTotal (API v3)|||ip
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml:686]: Let's use` !IP` (not only using VT)

Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml]: Done.
The task was changed and now it uses "!IP" automation.
Additionally, I separated the task from the VirusTotal section flow and added another section flow named "Check Reputation Using All Available Integrations"

File: Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml [hunk 2]
```diff
@@ -668,4 +814,6 @@ outputs:
   description: Endpoint domain name.
   type: string
 tests:
-- IP Enrichment - Generic v2 - Test
+- No tests (auto formatted)
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml:817]: Please add back the test playbook

Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml]: done

File: Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml [hunk 3]
```diff
@@ -315,6 +351,11 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml:355]: should be true

File: Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml [hunk 4]
```diff
@@ -633,17 +776,20 @@ inputs:
...
+  playbookInputQuery:
 - key: ResolveIP
   value:
     complex:
       root: inputs.ResolveIP
   required: true
   description: Whether to convert the IP address to a hostname using a DNS query (True/False).
+  playbookInputQuery:
 outputs:
 - contextPath: IP
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml:794]: lets add dbotscore as well

Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml:794]: The row below includes the DBotScore of all indicators...
 contextPath: DBotScore
 description: Indicator, Score, Type, and Vendor.

**Pull request merged**